pkg/agent/datapath/multiBridgeDatapath.go

```diff
@@ -184,7 +184,6 @@ type DpManager struct {
 	FlowIDToRules             map[uint64]*EveroutePolicyRuleEntry
 	flowReplayChan            chan struct{}
```

Comment:
nit: remove unused variable `flowReplayChan`

Reply:
flow replay event need to refactor, del this in other patch